kube/rbac.go

```diff
@@ -76,3 +101,40 @@ func NewRBACRole(name string, role model.AuthRole, settings ExportSettings) (hel
 
 	return container.Sort(), nil
 }
+
+// authPSP creates a block condition checking for RBAC and the named PSP
+func authPSP(psp string) helm.NodeModifier {
```

Comment:
Don't like the function name (given that there is also `authPSPRoleName`).

Maybe `authPSPCondition`?

Reply:
Like it. Will change.

Reply:
Changed with commit 30f483d98599f9a1db58c7720cd5882f4e189829